src/libs/renderer/src/screenshot.cpp

```diff
@@ -29,7 +29,7 @@ void DX9RENDER::SaveCaptureBuffers()
     for (fi = iCaptureFrameIndex; fi < iCaptureFrameIndex + 10000; fi++)
     {
         sprintf_s(cFileName, "k3cap_%04d.tga", fi);
-        if (_access(cFileName, 0) == -1)
+        if (!std::filesystem::exists(cFileName))
```

Comment:
it may throw. please do exception check or use overload:
```
bool exists( const std::filesystem::path& p, std::error_code& ec )
```

Reply:
Thanks. Done.